app/routes/projects/$projectId/editContributorsPath.tsx

```diff
@@ -0,0 +1,499 @@
+                                  Edit
+                                </Button>
+                              </Grid>
+                            )}
+                          </Grid>
+                        </Grid>
+                      </Grid>
+                    </Stack>
+                  ))}
+                  <Stack>
+                    <Grid
+                      container
+                      justifyContent="space-between"
+                      alignItems="flex-start"
+                    >
+                      {createTask === stage.id ? (
+                        <>
+                          <Grid item style={{ minWidth: "70%" }}>
+                            <ValidatedForm
+                              id="taskForm"
+                              validator={taskValidator}
+                              defaultValues={{
+                                id: "",
+                                description: "",
+                                position: stage.projectTasks.length + 1,
+                                projectStageId: stage.id,
+                              }}
+                              method="post"
+                              action="./editTask"
+                            >
+                              <input type="hidden" name="id" value="" />
+                              <input
+                                type="hidden"
+                                name="position"
+                                value={stage.projectTasks.length + 1}
+                              />
+                              <input
+                                type="hidden"
+                                name="projectStageId"
+                                value={stage.id}
+                              />
+                              <TextEditor
+                                name={`description`}
+                                label="Description"
+                                placeholder={"Describe the task..."}
+                              />
+                            </ValidatedForm>
+                          </Grid>
+                          <Grid item>
+                            <Grid
+                              container
+                              justifyContent="flex-end"
+                              alignItems="center"
+                              spacing={2}
+                              mt={1}
+                              mb={2}
+                            >
+                              <Grid item>
+                                <Button
+                                  type="submit"
+                                  form="taskForm"
+                                  variant="contained"
+                                  disabled={taskIsSubmitting}
+                                >
+                                  {taskIsSubmitting
+                                    ? "Saving..."
+                                    : "Save"}
```

Comment:
Also this one, could look better using just an icon but I don’t know what do you think